components/BidderProfile.tsx

```diff
@@ -0,0 +1,553 @@
+          <div className="extra-space" />
+          <div className="nouns">
+            {nouns &&
+              nounSVGs &&
+              nounSVGs.length > 0 &&
+              nounSVGs.map(
+                (nounSVG, index) =>
+                  nounSVG && (
+                    <Link
+                      href={
+                        nouns[index].id % 10 === 0
+                          ? `https://opensea.io/assets/ethereum/0x9c8ff314c9bc7f6e59a9d9225fb22946427edc03/${nouns[index].id}`
+                          : `/noun/${nouns[index].id}`
+                      }
+                      target="_blank"
+                      rel="noreferrer"
+                      key={index}
+                    >
+                      <img
+                        src={nounSVG}
+                        alt={`Noun ${nouns[index].id}`}
+                        style={{ paddingBottom: 'var(--s-2)', width: '24px' }}
+                      />
+                    </Link>
+                  ),
+              )}
+          </div>
+        </div>
+      </div>
+      <style jsx>{`
+        .dotted-hr {
+          height: 1px;
+          background-image: linear-gradient(to right, var(--lines) 90%, rgba(0, 0, 0, 0) 10%);
+          background-size: var(--s2) 1px;
+          margin: var(--s0) 0 var(--s1) 0;
+        }
+        .extra-space {
+          width: var(--s5);
+        }
+        .content {
+          margin: var(--s1) var(--s2) 0 var(--s1);
+        }
+        .external-link {
+          margin-left: var(--s-2);
+          color: var(--low-text);
+        }
+        .external-link:hover {
+          color: var(--bright-text);
+        }
+        .balance {
+          display: flex;
+          align-items: flex-start;
+          direction: column;
+          justify-content: left;
+        }
+        .nouns {
+          display: flex;
+          flex-wrap: wrap;
+          gap: var(${'--s-2'});
+          max-width: 50%;
+          height: 6rem;
+          overflow-y: auto;
+        }
+        .social-link:hover {
+          color: var(--bright-text);
+        }
+      `}</style>
+      <HorizontalLine />
+    </div>
+  );
+}
+
+function ProfileInfo({
+  wins,
+  bidderHistory,
+  address,
+  largestBid,
+}: {
+  wins: number;
+  bidderHistory: BidderHistory[] | undefined;
+  address: string;
+  largestBid:
+    | {
+        id: number;
+        value: string;
+        noun: Noun;
+      }
+    | undefined
+    | null;
+}) {
+  const totalBidsCount = bidderHistory?.reduce((acc, curr) => acc + Number(curr.countBids), 0) || 0;
+  const nounSVG = createNounSVG(largestBid?.noun as Noun);
+
+  return (
+    <div className="content">
+      <Stack direction="column" gap={1}>
+        <Text variant="title-1" bold color="low-text">
+          Bidder Strength
+        </Text>
+        <Text variant="title-2" bold color="low-text">
+          <Text variant="title-2" bold color="yellow">
+            {wins >= 3 ? '★★★' : wins >= 1 ? '★★' : '★'}
```

Comment:
I feel like we need to extract this into a function, something like estimating bidder strength. I'd say even if wins == 0 but the bidder has more than enough eth their strength should ++